common/changes/@stonecrop/beam/fix-ItemCount-slash_2025-01-08-18-16.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@stonecrop/beam",
+      "comment": "denominator condition to be shown",
+      "type": "none"
```

Minor nitpick, but the type should change to:

```suggestion
      "type": "patch"
```